docs/gcloud-auth.rst

```diff
@@ -58,15 +58,66 @@ properties of your local environment in the following order
 #. Credentials provided by the Google Cloud SDK (via ``gcloud auth login``)
 #. Application running in Google Compute Engine
 
-Loading Credentials Explicitly
-------------------------------
+Explicit Credentials
+====================
 
-In addition, the
+The Application Default Credentials discussed above can be useful
```

Comment:
> I have a hard time grokking what semantic line breaks are

I won't say the definition is crystal clear, but it seems pretty reasonable. Hit enter when you've written something that can stand, for the most part, on it's own.

```
                                                                                | (80-char line)
I won't say the definition is crystal clear,
but it seems pretty reasonable.
Hit enter when you've written something
that can stand, for the most part, on it's own.
```

Note that you could split the last line there into 3 lines if you really wanted.... Doesn't really matter to me...

Compared to:

```
                                                                                | (80-char line)
I won't say the definition is crystal clear, but it seems pretty reasonable. Hit
enter when you've written something that can stand, for the most part, on it's
own.
```

It seems kind of ... off .. that we're playing "hit enter only when you fill up the row" for version-controlled textual content.

---

> From where I'm sitting, it ain't broke, what are we fixing?

I'm not saying "go through and fix everything!". I'm saying we're adding new content, which is going to be version controlled... I think it's worthwhile. (http://programmer.97things.oreilly.com/wiki/index.php/The_Boy_Scout_Rule)